src/ImageSharp/Formats/Png/PngMetadata.cs

```diff
@@ -83,6 +84,11 @@ private PngMetadata(PngMetadata other)
     /// </summary>
     public uint RepeatCount { get; set; } = 1;
 
+    /// <summary>
+    ///  Gets or sets a value indicating whether the default image is shown as part of the animated sequence
+    /// </summary>
+    public bool DefaultImageAnimated { get; set; } = true;
```

Comment:
Let's call this `AnimateRootFrame`. That matches `ImageFrameCollection.RootFrame` naming convention.